### DISCUSSION TITLE
KTL Causing Login Issues???

### DISCUSSION THREAD

**Discussion by jbk68** [cortexrd/Knack-Toolkit-Library#115] [General]
Hey Norm, I'm getting a few customers calling me about login issues and it appears KTL may be causing it.  

With KTL installed, user tries to login in to a Knack app and the loading of page stops right before login screen is loaded.

If I remove and or disable (comment out)  KTL, apps load just fine but obviously I don't get all the awesome features that KTL brings.

Just a fyi, its not effecting users or computers that are logged in already just the ones trying to log in for the 1st time today.  I tried going to a couple of my apps on my tablet and I was experiencing the same log in issues.  Remove KTL, I'm able to login just fine.

Any suggestions???

**UPDATE: Reported login issues with Knack and here is there response:**

We found an error in the browser console referencing a KTL file and the problem appears to be this one https://ctrnd.s3.amazonaws.com/Lib/KTL/KTL_Start.js

Based on the error message, it appears that the file doesn't have defined the function of ktl.sysInfo.setCfg. Temporarily commenting your JS code should fix this problem, while Norm finds what happened to their code.

**Comment by cortexrd** [comment]
Indeed, accidentally released a wrong file this afternoon.

I've released a fix: **KTL v0.16.3**.

Terribly sorry about that.  I will put in place some additional measures to make sure doesn't happen again.

Norm
